rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -118,18 +118,18 @@ namespace rsx
 				image.set_raw_image(icon_info.get());
 			}
 
-			std::string trophy_message;
+			localized_string_id string_id = localized_string_id::INVALID;
 			switch (trophy.trophyGrade)
 			{
-			case SCE_NP_TROPHY_GRADE_BRONZE: trophy_message = "bronze"; break;
-			case SCE_NP_TROPHY_GRADE_SILVER: trophy_message = "silver"; break;
-			case SCE_NP_TROPHY_GRADE_GOLD: trophy_message = "gold"; break;
-			case SCE_NP_TROPHY_GRADE_PLATINUM: trophy_message = "platinum"; break;
+			case SCE_NP_TROPHY_GRADE_BRONZE: string_id = localized_string_id::RSX_OVERLAYS_TROPHY_BRONZE; break;
+			case SCE_NP_TROPHY_GRADE_SILVER: string_id = localized_string_id::RSX_OVERLAYS_TROPHY_SILVER; break;
+			case SCE_NP_TROPHY_GRADE_GOLD: string_id = localized_string_id::RSX_OVERLAYS_TROPHY_GOLD; break;
+			case SCE_NP_TROPHY_GRADE_PLATINUM: string_id = localized_string_id::RSX_OVERLAYS_TROPHY_PLATINUM; break;
 			default: break;
 			}
 
-			trophy_message = "You have earned the " + trophy_message + " trophy\n" + trophy.name;
-			text_view.set_text(trophy_message);
+			text_view.set_text(trophy.name);
+			text_view.set_text(get_localized_u32string(string_id) + text_view.text);
```

Comment:
This doesn't appear translatable.
The translator won't know the position of the object within the sentence.  Translatable strings should NEVER be concatenated like this.

Reply:
you don't say. That's what I said in the OP